src/components/SkiCard.tsx

```diff
@@ -10,7 +10,7 @@ const SkiCard = ({ result }: CardProps<Ce_skis>) => {
   const terrain = skis.c_terrain?.[0];
 
   return (
-    <Link href={skis.slug}>
+    <Link href={`/${skis.slug}`}>
```

Comment:
does this work? We need the exact modification of the slug field in the KG to be documented.

Reply:
The slugs for the Skis are formed as follows: [[c_genderName]]/[[c_productType]]/[[c_categoryName]]/[[entityId]]

If I don't add the slash, I am directed to a page like /men/men/skis/powder/ski_1. The slash makes the navigation work.